### ISSUE TITLE
[CLI]: Stop saving system information to save memory

### ISSUE THREAD

**Issue by Kartik-Neralwar** [wandb/wandb#5990]
### Describe the bug

I use wandb callback with Keras. I am trying to only save the important metrics like loss, and validation loss. I want wandb to not save the system information e.g. Process GPU Power Usage (W), to save memory. I have tried setting save_model=False and save_graph=False, but wandb still save the system information. 

```python

```
        model.fit(gen.flow(x_train_new, y_train_new, batch_size=batch_size),
                           steps_per_epoch=x_train.shape[0]/ batch_size,
                           epochs=epochs,
                           verbose=verbose,
                           callbacks=[csv_logger, checkpoint,change_lr,early_stop, WandbCallback(save_model=False, save_graph=False)],
                           validation_data=gen.flow(x_val_new, y_val_new, batch_size=batch_size),
                           use_multiprocessing=True,
                           workers=18)

```shell

```

### Environment

WandB version:0.14.2

OS:Ubuntu 20

Python version:3.9.7

Versions of relevant libraries:

**Comment by umakrishnaswamy** [comment]
Hey @Kartik-Neralwar, we have a  `_disable_stats: Optional[bool]` setting that you can define in your wandb.init call. Your setup should look something similar to the following:

`wandb.init(project='turn_off_sys_metrics', settings=wandb.Settings(_disable_stats=True, _disable_meta=True))
`

Then you can run your training as expected.

**Comment by Kartik-Neralwar** [comment]
Thank you @umakrishnaswamy!
